_im-plugin/index-rollups/index.md

```diff
@@ -524,23 +524,38 @@ The `doc_count` field in bucket aggregations contains the number of documents co
 .nobr { white-space: nowrap }
 </style>
 
-In ISM rollup, the `target_index` field may contain a template that is compiled at the time of each rollup indexing. For example, if you specify the `target_index` field as <span style="white-space: nowrap">`{% raw %}rollup_ndx-{{ctx.source_index}}{% endraw %}`,</span> the source index `log-000001` will roll up into a target index `rollup_ndx-log-000001`. This allows you to roll up data into multiple time-based indices, with one rollup job created for each source index. 
+In ISM rollup, the `target_index` field may contain a template that is compiled at the time of each rollup indexing. For example, if you specify the `target_index` field as <span style="white-space: nowrap">`{% raw %}rollup_ndx-{{ctx.source_index}}{% endraw %}`,</span> the source index `log-000001` will roll up into a target index `rollup_ndx-log-000001`. This allows you to roll up data into multiple time-based indexes, with one rollup job created for each source index. 
 
 The `source_index` parameter in {% raw %}`{{ctx.source_index}}`{% endraw %} cannot contain wildcards.
 {: .note}
 
-## Searching multiple rollup indices
+## Query strings
 
-When data is rolled up into multiple target indices, you can run one search across all of the rollup indices. To search multiple target indices that have the same rollup, specify the index names as a comma-separated list or a wildcard pattern. For example, with `target_index` as <span style="white-space: nowrap">`{% raw %}rollup_ndx-{{ctx.source_index}}{% endraw %}`</span> and source indices that start with `log`, specify the `rollup_ndx-log*` pattern. Or, to search for rolled up log-000001 and log-000002 indices, specify the `rollup_ndx-log-000001,rollup_ndx-log-000002` list.
+To take advantage of shorter and easier to write strings in Query DSL, you can use [query strings]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text/#query-string) to simplify search queries in rollup indexes. To use query strings, add the following fields to your rollup search request.
 
-You cannot search a mix of rollup and non-rollup indices with the same query.
+```json
+"query": {
+      "query_string": {
+          "query": "field_name:field_value"
+      }
+  }
+```
+
+For more information on what fields are supported in query strings, see [Advanced filter options]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text/#advanced-filter-options/).
```

Comment:
Unfortunately, we don't yet have a doc page for query_string query. And the parameter list in Advanced filter options is not 1:1. For example, query_string query does not support `low_freq_operator`. I don't know what the best way of dealing with this is: probably prioritize query_string query documentation and then link it here?

Reply:
I'll go ahead and add it since I have a couple more days. For now, we could add a warning.